src/app/views/query-runner/request/permissions/Permission.tsx

```diff
@@ -60,7 +63,8 @@ export class Permission extends Component<IPermissionProps, IPermissionState> {
       || nextProps.consentedScopes !== this.props.consentedScopes
       || nextProps.dimensions !== this.props.dimensions
       || nextProps.permissionsPanelOpen !== this.props.permissionsPanelOpen
-      || nextState.permissions !== this.state.permissions;
+      || nextState.permissions !== this.state.permissions
+      || nextProps.permissionModeType !== this.props.permissionModeType
```

Comment:
```suggestion
      || nextProps.permissionModeType !== this.props.permissionModeType;
```
Line 63 had the semicolon but line 67 forgot it ^^

Reply:
(´＠＿＠｀ ) great catch!

Reply:
resolved in https://github.com/LokiLabs/microsoft-graph-explorer-v4/pull/16/commits/55666db4d0964581599d5f907b355fb066034d67